client/src/main.rs

```diff
@@ -117,16 +117,16 @@ fn main() {
 						.default_value("2000")
 						.help("worker direct invocation port"),
 				)
-				.name("substratee-client")
+				.name("integritee-cli")
 				.version(VERSION)
 				.author("Supercomputing Systems AG <info@scs.ch>")
```

Comment:
this should now be Integritee AG

Reply:
hello@integritee.network

Reply:
I suppose this change should be made for all Supercomputing authors.